test/MassMigration.test.ts

```diff
@@ -14,6 +14,7 @@ import { Result } from "../ts/interfaces";
 import { Tree } from "../ts/tree";
 import { mineBlocks } from "../ts/utils";
 import { expectRevert } from "../ts/utils";
+import { constants } from "ethers";
 
 describe("Mass Migrations", async function() {
```

Comment:
I intend to replace the whole `test/rollup.test.ts`, `test/MassMigration.test.ts`, and `test/rollupCreate2Transfer.test.ts` with the new integration test. So I only attempt to fix the first test, but bandaid the latter two tests with dummy genesis roots.